README.md

```diff
@@ -0,0 +1,155 @@
+
+# Fuel Telemetry
+
+`fuel-telemetry` is a `tracing` `Layer` used to implement telemetry within our
+apps and libraries.
+
+Steps to get `fuel-telemetry` going:
+
+- Create a `TelemetryLayer` `tracing` `Layer`
+- Add the `TelemetryLayer` to a `tracing` `Subscriber`
+- Enable the `telemetry` field on `Span`s you want sent to InfluxDB
+- Use the regular `tracing` macros (e.g `info!()`, `warn!()` etc) to record `Events` within `Span`s
+- Create a `FileWatcher` so that telemetry files will be sent to InfluxDB
+
+Most of these steps are hidden away behind convenience functions within `TelemetryLayer`.
+
+## Using `fuel-telemetry`
+
+### Within Applications with No Existing Tracing (Enabled by Default)
+
+If you're writing an app where:
+
+- `fuel-telemetry` will be your only `tracing` `Layer` and `Subscriber`
+- and **you want all** `tracing` `Event`s to be sent to InfluxDB by default
+
+then the quickest way to get going is:
+
+    use fuel_telemetry::prelude::*;
+
+    fn main() {
+        telemetry_init().unwrap();
+
+        info!("This event will be sent to InfluxDB");
+    }
```

Comment:
wrap in code block:
```suggestion
```rust
    use fuel_telemetry::prelude::*;

    fn main() {
        telemetry_init().unwrap();

        info!("This event will be sent to InfluxDB");
    }
```
```
^ closing "```" (not shown in suggestion above)

Reply:
needs to be applied to all code blocks in readme.